### DISCUSSION TITLE
undomanager.add()

### DISCUSSION THREAD

**Discussion by yashvi2026** [GrapesJS/grapesjs#5636] [Q&A]
What should be passed to undomanager.add(),Itried passing editor.getSelected((),editor.UndoManager.getStack() but nothing works .same for undomanager.remove()?

**Comment by pcjaarvis** [comment]
You have to pass the model which u want to track, or remove tracking changes for.
[Docs Link](https://grapesjs.com/docs/api/undo_manager.html#add)
